db/migrate/20190123151514_add_application_purpose_to_nerits_assessments.rb

```diff
@@ -0,0 +1,5 @@
+class AddApplicationPurposeToNeritsAssessments < ActiveRecord::Migration[5.2]
```

Comment:
the name of this migration refers to 'Nerits' rather than 'Merits' not a huge issue at all and maybe more work to change than is worth